src/app/opportunities-research-partners/opportunities-research-partners-page.component.scss

```diff
@@ -32,7 +32,7 @@
   h1 {
     font-size: 34px;
     font-weight: 700;
-    line-height: math.div(27, 34);
+    // line-height: math.div(27, 34);
```

Comment:
Don't comment out... delete if no longer applicable. I think case, I think it's _still_ needed.

Reply:
Setting it to that makes the text overlap. I think it would have to update the values a little

Reply:
Cool but if it didn't overflow overlap before, why is it overlapping now?

Reply:
Because the text now spans more than one line

Reply:
setting it to `math.div(27, 27)` seems to be a good spot for both mobile and desktop

Reply:
No content for the `opportunities` page, should I just remove the lorem ipsum text?

Reply:
> No content for the `opportunities` page, should I just remove the lorem ipsum text?

@kilemensi this is related to #24